packages/story-editor/src/components/publishModal/mainContent/mandatoryStoryInfo.js

```diff
@@ -80,7 +88,8 @@ const MandatoryStoryInfo = ({
             'Stories with a description tend to do better on search and have a wider reach',
             'web-stories'
           )}
-          onChange={handleUpdateStoryInfo}
+          onChange={onInputChange}
+          onBlur={handleUpdateStoryInfo}
```

Comment:
Think we'll just have to update the unit tests here https://github.com/GoogleForCreators/web-stories-wp/runs/5242885436?check_suite_focus=true